service/src/test/java/uk/gov/hmcts/reform/fpl/service/GatekeeperEmailContentProviderTest.java

```diff
@@ -22,16 +26,29 @@
 import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.populatedCaseDetails;
 
 @ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {JacksonAutoConfiguration.class, GatekeeperEmailContentProvider.class,
+    DateFormatterService.class, HearingBookingService.class})
 class GatekeeperEmailContentProviderTest {
 
     private static final String LOCAL_AUTHORITY_CODE = "example";
 
-    @Mock
+    @MockBean
     private LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration;
 
-    @InjectMocks
+    @Autowired
+    private ObjectMapper mapper;
+
+    private final DateFormatterService dateFormatterService = new DateFormatterService();
+    private final HearingBookingService hearingBookingService = new HearingBookingService();
+
```

Comment:
given your adding both _HearingBookingService_ and _DateFormatterService_ to _ContextConfiguration_ why are we instantiating instead of _@Autowired_ ?